flask_monitoringdashboard/controllers/endpoints.py

```diff
@@ -144,10 +144,11 @@ def set_endpoint_rule(session, endpoint_name, monitor_level):
     """
     update_endpoint(session, endpoint_name, value=monitor_level)
 
-    # Remove wrapper
-    original = getattr(config.app.view_functions[endpoint_name], 'original', None)
-    if original:
-        config.app.view_functions[endpoint_name] = original
-    session.commit()
-
-    add_decorator(get_endpoint_by_name(session, endpoint_name))
+    if config.app.view_functions.get(endpoint_name):
+        # Remove wrapper
+        original = getattr(config.app.view_functions[endpoint_name], 'original', None)
+        if original:
+            config.app.view_functions[endpoint_name] = original
+        session.commit()
```

Comment:
this line was already in the old code, but I think that's incorrect, since we're not making database changes, so committing has no effect. Can you remove this line?

Reply:
Yes I will!